modules/delegates/components/modals/UndelegateModal.tsx

```diff
@@ -44,7 +44,7 @@ export const UndelegateModal = ({
   const voteDelegateAddress = delegate.voteDelegateAddress;
   const [mkrToWithdraw, setMkrToWithdraw] = useState(BigNumber.from(0));
 
-  const { data: mkrStaked } = useMkrDelegated(account, voteDelegateAddress);
+  const { data: mkrStaked } = useMkrDelegatedByUser(account, voteDelegateAddress);
```

Comment:
Can the user undelegate the MKR delegated outside the gov portal?

Reply:
Good catch, will update this in another PR to only show the amount delegated directly, and add to the disclaimer info about the seal module